docs/usage/parameters.md

```diff
@@ -27,5 +27,9 @@ The following parameters are currently available:
 - `file`: Name of the TXT carbon footprint report containing the energy consumption, the estimated CO<sub>2</sub> emission and other relevant metrics for each task.
 - `reportFile`: Name of the HTML report containing information about the entire carbon footprint, overview plots and more detailed task-specific metrics.
 - `ci`: carbon intensity of the respective energy production.
+- `location`: location code to automatically retrieve a location-specific CI value.
+For countries, these are [ISO alpha-2 codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2). 
+For regions, it’s the ISO alpha-2 code for the country, followed by an identifier for the state, e.g. US-CA for “California, USA”.
+You can find the available data [here](../../plugins/nf-co2footprint/src/resources/CI_aggregated.v2.2.csv).
```

Comment:
Would it be worth clarifying here that either `ci` or `location` should be used, and if both are specified, which one takes over? (It should probably be `ci`). (Sorry if this is done elsewhere!)
e.g. :
- adding "Only used if CI is not specified" to the description of `location` 
- and adding to the description of `ci ` above: "If not specified, `location` should be used"

Reply:
Ah I see below that having both raises an error!

Reply:
I added a note that they are mutually exclusive now